code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -304,6 +305,7 @@
 			module_sprites["Eve"] = "eve"
 			module_sprites["Sleek"] = "sleekmedic"
 			module_sprites["#17"] = "servbot-medi"
+			module_sprites["Arachne"] = "aracne"
```

Comment:
Might be worth changing for the sake  of consistency before someone does a "spelling fix" and breaks this sprite.